src/ucp/api/ucp.h

```diff
@@ -795,6 +795,8 @@ void ucp_ep_destroy(ucp_ep_h ep);
  * prior to this call are completed both at the origin and at the target
  * @ref ucp_ep_h "endpont" when this call returns.
  *
+ * @note Errors originating from non-blocking calls may return from invocation.
```

Comment:
The note does not read right. I would suggest something like:
`The call flushes outstanding communication operations and therefore it may return error codes associated with the operations. `

Does it sound right ?